docs/examples/frameworks/pytorch_setup/README.rst

```diff
@@ -27,6 +27,34 @@ PyTorch Setup
 
 **Running this example**
 
+This assumes that you already created a conda environment named "pytorch".
+To create this environment, we first request resources for an interactive job.
+Note that we are requesting a GPU for this job, even though we're only going to install packages.
+This is because we want PyTorch to be installed with GPU support, and to have all the required
+libraries.
```

Comment:
Width to 80 char
```suggestion
This assumes that you already created a conda environment named "pytorch". To
create this environment, we first request resources for an interactive job.
Note that we are requesting a GPU for this job, even though we're only going to
install packages. This is because we want PyTorch to be installed with GPU
support, and to have all the required libraries.
```